lib/matplotlib/tests/test_font_manager.py

```diff
@@ -96,3 +97,29 @@ def test_hinting_factor(factor):
     # Check that hinting only changes text layout by a small (10%) amount.
     np.testing.assert_allclose(hinted_font.get_width_height(), expected,
                                rtol=0.1)
+
+
+def test_user_fonts():
+    if not os.environ.get('APPVEYOR', False):
+        pytest.xfail('This test does only work on appveyor since user fonts '
+                     'are Windows specific')
```

Comment:
Not only that, but you don't even want to run it on a dev's Windows computer to not overwrite their fonts directory.

Reply:
Yes, that it true. I agree that respecting the user directories of other developers is a thing to keep in mind. Do you want me to change the message or was this only a mere side note?^^

Reply:
Please do change the note too.